src/scripts/ibmcloud-auth.js

```diff
@@ -111,6 +127,40 @@ if (env.readerUsers) {
 	READER_EMAILS = env.readerUsers.split(',');
 }
 
+function ldapInit() {
+	var promise = new Promise((resolve, reject) => {
+		if (env.ldapServer && env.ldapPort && env.ldapBindUser && env.ldapBindPassword) {
+			var ldapUrl = `${env.ldapProtocol}://${env.ldapServer}:${env.ldapPort}`;
+			POWER_GROUPS = env.ldapPowerGroups.split(';');
```

Comment:
LDAP users have commas in the distinguished names: `ou=mathematicians,dc=example,dc=com`